database/migration/migrateData.ts

```diff
@@ -1426,14 +1426,90 @@ const migrateData = async () => {
         ALTER COLUMN reviewability
           SET DEFAULT 'ONLY_IF_APPLICANT_ANSWER';`)
 
-
     console.log(' - Add timestamp and email_server_log to notification table')
     await DB.changeSchema(`
       ALTER TABLE notification 
         ADD COLUMN IF NOT EXISTS timestamp timestamptz
           DEFAULT CURRENT_TIMESTAMP NOT NULL,
         ADD COLUMN IF NOT EXISTS email_server_log varchar; 
     `)
+
+    console.log('add unique constraint to usernames in database')
```

Comment:
Minor nit -- can you format the console text the same as the others in this file?